JPS_Ontology/ontology/ontodevice/OntoDevice.csv

```diff
@@ -140,18 +140,18 @@ OperatingProperty,Class,,,,,,A property of an operating parameter of a device (e
 OperatingRange,Class,OperatingProperty,IS-A,,,,The range for an operating parameter of a device (e.g. range of an operating temperature).,https://www.theworldavatar.com/kg/ontodevice/,Operating range
 OperatingTemperatureRange,Class,OperatingRange,IS-A,,,,The operating temperature range of a device.,https://www.theworldavatar.com/kg/ontodevice/,Operating temperature range
 Activity,Class,,,,,,An event that occurs over a period of time and involves specific actions such as walking or running.,https://www.theworldavatar.com/kg/ontodevice/,Activity
-ActivityType,Class,,,,,,Represents the type of activity being performed (e.g., walking, biking).,https://www.theworldavatar.com/kg/ontodevice/,Activity Type
-OnFoot,Class,ActivityType,IS-A,,,,The device is on a user who is walking or running.,http://www.theworldavatar.com/kg/ontodevice/,On Foot
-InVehicle,Class,ActivityType,IS-A,,,,The device is in a vehicle.,http://www.theworldavatar.com/kg/ontodevice/,In Vehicle
-OnBicycle,Class,ActivityType,IS-A,,,,The device is on a bicycle.,http://www.theworldavatar.com/kg/ontodevice/,On Bicycle
+ActivityType,Class,,,,,,Represents the type of activity being performed (e.g.\, walking\, biking).,https://www.theworldavatar.com/kg/ontodevice/,Activity type
+OnFoot,Class,ActivityType,IS-A,,,,The device is on a user who is walking or running.,http://www.theworldavatar.com/kg/ontodevice/,On foot
+InVehicle,Class,ActivityType,IS-A,,,,The device is in a vehicle.,http://www.theworldavatar.com/kg/ontodevice/,In vehicle
+OnBicycle,Class,ActivityType,IS-A,,,,The device is on a bicycle.,http://www.theworldavatar.com/kg/ontodevice/,On bicycle
 Running,Class,OnFoot,IS-A,,,,A sub-activity of on foot where the user is running.,http://www.theworldavatar.com/kg/ontodevice/,Running
 Walking,Class,OnFoot,IS-A,,,,A sub-activity of on foot where the user is walking.,http://www.theworldavatar.com/kg/ontodevice/,Walking
-ConfidenceLevel,Class,,,,,,Represents the confidence level indicating the likelihood of the activity being accurate.,https://www.theworldavatar.com/kg/ontodevice/,Confidence Level
+Confidence level,Class,,,,,,Represents the confidence level indicating the likelihood of the activity being accurate.,https://www.theworldavatar.com/kg/ontodevice/,Confidence Level
```

Comment:
For values under the Source column, there should not be any spaces, it should be ConfidenceLevel instead of Confidence level.

The label for this class is incorrect. It should be Confidence level instead of Confidence Level.